Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Umi-OCR/UI](https://hosted.weblate.org/projects/umi-ocr/ui/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/umi-ocr/ui/horizontal-auto.svg)

Commit:
Translated using Weblate (French)

Currently translated at 42.2% (192 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/fr/

rashbip committed:
Added translation using Weblate (Bengali)

rashbip committed:
Translated using Weblate (Bengali)

Currently translated at 3.5% (16 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/bn/

ChouChiu committed:
Translated using Weblate (Chinese (Traditional Han script))

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/zh_Hant/

safe1ine commented:
@monkeycode-ai review 一下

MonkeyCode-AI commented:
MonkeyCode-AI 正在分析任务...

MonkeyCode-AI commented on dev-tools/i18n/release/bn.ts:
```diff
@@ -0,0 +1,2482 @@
+    <message>
+        <location filename="../../../UmiOCR-data/qt_res/qml/TabPages/About/About.qml" line="117"/>
+        <source>复制</source>
+        <translation type="unfinished"></translation>
```
> [!WARNING]
> **⚠️** 新增孟加拉语翻译文件仍包含大量 `type="unfinished"`，可能导致运行时回退/显示空字符串
> 
> bn.ts 为新增语言文件，但从 diff 可见多处 `<translation type="unfinished"></translation>`（例如 About/复制、AsynFilesLoader/正在载入...、BatchDOC/加密等）。在 Qt Linguist/TS 体系中，unfinished 往往会导致 UI 回退到 sourceLanguage 或显示为空（取决于加载与回退策略）。如果该目录属于 release 发布资源，未完成翻译会直接影响最终用户体验与一致性。
> 
> **建议**: 建议在 release 目录中仅放入达到最低完成度阈值的 TS，或在构建/发布脚本中对 `type="unfinished"` 做检测并在 CI 阶段提示/阻断；若允许未完成翻译，也应确保回退策略明确（例如回退到 en/zh_CN 而非空）。
>

MonkeyCode-AI commented on dev-tools/i18n/release/bn.ts:
```diff
@@ -0,0 +1,2482 @@
+    </message>
+    <message>
+        <location filename="../../../UmiOCR-data/qt_res/qml/TabPages/About/About.qml" line="86"/>
+        <source>译者</source>
```
> [!WARNING]
> **⚠️** 存在明显未翻译/错误语言内容（译者字段仍为中文“译者”）
> 
> 在 About context 中，`<source>译者</source>` 的 translation 仍是 `译者`（中文），这属于明显的本地化错误，且与同文件其他条目风格不一致。类似问题如果存在于多个字段，会降低本地化质量。
> 
> **建议**: 将该条目翻译为孟加拉语（例如“অনুবাদক”或更符合项目术语的译法），并建议对 bn.ts 做一次快速扫描：查找 translation 与 source 完全相同的非专有名词条目。
>

MonkeyCode-AI commented on dev-tools/i18n/release/fr_FR.ts:
```diff
@@ -1399,12 +1400,12 @@ Do you want to end the task and close the page?</translation>
         <location filename="../../../UmiOCR-data/qt_res/qml/Widgets/MissionCtrlPanel.qml" line="222"/>
         <source>正在保存</source>
-        <translation type="unfinished"></translation>
+        <translation>Entregistrement</translation>
```
> [!TIP]
> **💡** 法语翻译存在拼写问题："Entregistrement"
> 
> 在 MissionCtrlPanel 的 `正在保存` 翻译为 `Entregistrement`，法语正确拼写应为 `Enregistrement`。此类拼写错误会直接暴露在 UI 文案中。
> 
> **建议**: 将 `Entregistrement` 更正为 `Enregistrement`。
>

MonkeyCode-AI commented on dev-tools/i18n/release/fr_FR.ts:
```diff
@@ -1212,7 +1213,7 @@ Do you want to end the task and close the page?</translation>
         <location filename="../../../UmiOCR-data/qt_res/qml/Widgets/ImageViewer/ImageScale.qml" line="74"/>
         <source>打开图片</source>
-        <translation type="unfinished"></translation>
+        <translation>Ouvrir image</translation>
```
> [!TIP]
> **💡** 法语翻译风格不一致：缺少冠词/介词（"Ouvrir image"）
> 
> `打开图片` 被翻译为 `Ouvrir image`，法语更自然的表达通常是 `Ouvrir l'image`（本文件中其他位置也使用了 `Ouvrir l&apos;image`）。同一含义在同文件内的译法不一致可能导致 UI 文案不统一。
> 
> **建议**: 建议统一为 `Ouvrir l'image`（与该 PR 里 BatchOCR 的 `Ouvrir l&apos;image` 保持一致）。
>

MonkeyCode-AI commented on dev-tools/i18n/release/zh_TW.ts:
```diff
@@ -1039,7 +1039,7 @@
         <location filename="../../../UmiOCR-data/qt_res/qml/Configs/GlobalConfigs.qml" line="246"/>
         <source>操作</source>
-        <translation type="unfinished">操作</translation>
+        <translation>操作</translation>
```
> [!TIP]
> **💡** 繁中修正仅移除 unfinished 标记，建议确认是否有发布门禁依赖该标记
> 
> zh_TW.ts 的变更把 `type="unfinished"` 去掉但译文不变。这通常是期望行为（标记翻译完成）。但如果项目的翻译统计/发布门禁依赖 unfinished 状态（例如自动检查完成率），需确认移除 unfinished 不会误标“已人工校对”。
> 
> **建议**: 若流程上允许自动移除 unfinished，建议在 PR/CI 中增加“由 Weblate 自动更新”的注释或在发布前进行抽样校对；否则保留 unfinished 直到人工审核。
>

MonkeyCode-AI reviewed (commented):
> 我是 [MonkeyCode AI 编程助手](https://monkeycode-ai.com/)，你可以在 GitHub 仓库的 PR 中 at @MonkeyCode-AI 来呼唤我。

> 任务执行细节请参考: [https://monkeycode-ai.com](https://monkeycode-ai.com/tasks/public?id=f93149e4-361a-478b-94ff-e1cffb4ac648)

## 代码审查结果

本次 PR 主要为 UI 增加孟加拉语翻译文件并补全法语、繁中部分翻译；整体为非逻辑改动，但新增 bn.ts 中仍包含大量 unfinished/未翻译条目，且存在明显未本地化文本，建议在发布流程中拦截或明确允许。

| 🚨 Critical | ⚠️ Warning | 💡 Suggestion |
|:-:|:-:|:-:|
| 0 | 2 | 3 |

yigitcilb committed:
Added translation using Weblate (Turkish)

Commit:
Translated using Weblate (Chinese (Traditional Han script))

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/zh_Hant/

Commit:
Translated using Weblate (German)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/de/

Commit:
Translated using Weblate (Russian)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/ru/

Commit:
Translated using Weblate (Tamil)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/ta/

Commit:
Translated using Weblate (Nepali)

Currently translated at 34.5% (157 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/ne/

Commit:
Translated using Weblate (French)

Currently translated at 50.4% (229 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/fr/

Commit:
Translated using Weblate (Uzbek)

Currently translated at 1.9% (9 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/uz/

Commit:
Translated using Weblate (Persian)

Currently translated at 3.9% (18 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/fa/

Commit:
Translated using Weblate (Kabyle)

Currently translated at 12.7% (58 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/kab/

Commit:
Translated using Weblate (Hebrew)

Currently translated at 9.1% (41 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/he/

Commit:
Translated using Weblate (Korean)

Currently translated at 5.3% (24 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/ko/

Commit:
Translated using Weblate (Vietnamese)

Currently translated at 8.1% (37 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/vi/

Commit:
Translated using Weblate (Spanish)

Currently translated at 32.2% (145 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/es/

Commit:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/pt/

Commit:
Translated using Weblate (Japanese)

Currently translated at 90.7% (412 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/ja/

Commit:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/pt/

Commit:
Translated using Weblate (French)

Currently translated at 50.6% (230 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/fr/

Commit:
Translated using Weblate (Hebrew)

Currently translated at 9.3% (42 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/he/

Commit:
Translated using Weblate (Spanish)

Currently translated at 33.5% (151 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/es/

Commit:
Translated using Weblate (English (United States))

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en_US/

Commit:
Translated using Weblate (Japanese)

Currently translated at 96.2% (437 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/ja/

Commit:
Translated using Weblate (English (United States))

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en_US/

Commit:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/pt/

Commit:
Translated using Weblate (French)

Currently translated at 52.4% (238 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/fr/

Commit:
Translated using Weblate (Japanese)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/ja/

Commit:
Translated using Weblate (Hebrew)

Currently translated at 11.1% (50 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/he/

Commit:
Translated using Weblate (Spanish)

Currently translated at 36.6% (165 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/es/

Commit:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/pt/

Commit:
Translated using Weblate (Spanish)

Currently translated at 36.6% (165 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/es/

Commit:
Translated using Weblate (Russian)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/ru/

Commit:
Translated using Weblate (German)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/de/

Commit:
Translated using Weblate (Kabyle)

Currently translated at 12.7% (58 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/kab/

Commit:
Translated using Weblate (Chinese (Traditional Han script))

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/zh_Hant/

Commit:
Translated using Weblate (Tamil)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/ta/

Commit:
Translated using Weblate (Hebrew)

Currently translated at 11.3% (51 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/he/

Commit:
Translated using Weblate (French)

Currently translated at 56.8% (258 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/fr/

Commit:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/pt/

Commit:
Translated using Weblate (Hebrew)

Currently translated at 18.2% (82 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/he/

Commit:
Translated using Weblate (Spanish)

Currently translated at 43.5% (196 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/es/

Commit:
Translated using Weblate (English (United States))

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en_US/

Commit:
Translated using Weblate (French)

Currently translated at 61.0% (277 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/fr/

Commit:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/pt/

Commit:
Translated using Weblate (Hebrew)

Currently translated at 25.5% (115 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/he/

Commit:
Translated using Weblate (Spanish)

Currently translated at 50.8% (229 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/es/

Commit:
Translated using Weblate (Arabic)

Currently translated at 2.2% (10 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/ar/

Commit:
Translated using Weblate (English (United States))

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en_US/

Commit:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/pt/

Commit:
Translated using Weblate (Hebrew)

Currently translated at 27.7% (125 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/he/

Commit:
Translated using Weblate (Spanish)

Currently translated at 53.1% (239 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/es/

Commit:
Translated using Weblate (French)

Currently translated at 62.5% (284 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/fr/

Commit:
Translated using Weblate (Korean)

Currently translated at 5.7% (26 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/ko/

Commit:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/pt/

Commit:
Translated using Weblate (French)

Currently translated at 65.8% (299 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/fr/

Commit:
Translated using Weblate (Hebrew)

Currently translated at 35.3% (159 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/he/

Commit:
Translated using Weblate (Spanish)

Currently translated at 60.6% (273 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/es/

Commit:
Translated using Weblate (German)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/de/

Commit:
Translated using Weblate (Kabyle)

Currently translated at 12.7% (58 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/kab/

Commit:
Translated using Weblate (Russian)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/ru/

Commit:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/pt/

Commit:
Translated using Weblate (Spanish)

Currently translated at 62.8% (283 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/es/

Commit:
Translated using Weblate (Hebrew)

Currently translated at 39.3% (177 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/he/

Commit:
Translated using Weblate (French)

Currently translated at 68.2% (310 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/fr/

Commit:
Added translation using Weblate (English)

Commit:
Translated using Weblate (Spanish)

Currently translated at 62.8% (283 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/es/

Commit:
Translated using Weblate (English)

Currently translated at 1.1% (5 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en/

Commit:
Translated using Weblate (Hebrew)

Currently translated at 43.1% (194 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/he/

Commit:
Translated using Weblate (French)

Currently translated at 70.9% (322 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/fr/

Commit:
Translated using Weblate (French)

Currently translated at 75.7% (344 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/fr/

Commit:
Translated using Weblate (English (United States))

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en_US/

Commit:
Translated using Weblate (English)

Currently translated at 5.0% (23 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en/

Commit:
Translated using Weblate (Hebrew)

Currently translated at 49.5% (223 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/he/

Commit:
Translated using Weblate (Spanish)

Currently translated at 65.1% (293 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/es/

Commit:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/pt/

Commit:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/pt/

Commit:
Translated using Weblate (French)

Currently translated at 77.3% (351 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/fr/

Commit:
Translated using Weblate (Spanish)

Currently translated at 67.3% (303 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/es/

Commit:
Translated using Weblate (English)

Currently translated at 7.4% (34 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en/

Commit:
Translated using Weblate (English (United States))

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en_US/

Commit:
Translated using Weblate (Hebrew)

Currently translated at 52.4% (236 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/he/

Commit:
Translated using Weblate (Hebrew)

Currently translated at 52.6% (237 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/he/

Commit:
Translated using Weblate (French)

Currently translated at 78.4% (356 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/fr/

Commit:
Translated using Weblate (English)

Currently translated at 8.1% (37 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en/

Commit:
Translated using Weblate (Russian)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/ru/

Commit:
Translated using Weblate (English (United States))

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en_US/

Commit:
Translated using Weblate (Spanish)

Currently translated at 69.7% (314 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/es/

Commit:
Translated using Weblate (French)

Currently translated at 81.2% (369 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/fr/

Commit:
Translated using Weblate (Hebrew)

Currently translated at 56.2% (253 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/he/

Commit:
Translated using Weblate (English)

Currently translated at 11.6% (53 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en/

Commit:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/pt/

Commit:
Translated using Weblate (English (United States))

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en_US/

Commit:
Translated using Weblate (French)

Currently translated at 81.9% (372 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/fr/

Commit:
Translated using Weblate (Hebrew)

Currently translated at 57.5% (259 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/he/

Commit:
Translated using Weblate (English)

Currently translated at 12.9% (59 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en/

Commit:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/pt/

Commit:
Translated using Weblate (Spanish)

Currently translated at 71.1% (320 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/es/

Commit:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/pt/

Commit:
Translated using Weblate (English (United States))

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en_US/

Commit:
Translated using Weblate (French)

Currently translated at 84.8% (385 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/fr/

Commit:
Translated using Weblate (Hebrew)

Currently translated at 62.4% (281 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/he/

Commit:
Translated using Weblate (English)

Currently translated at 17.8% (81 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en/

Commit:
Translated using Weblate (Spanish)

Currently translated at 76.0% (342 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/es/

Commit:
Translated using Weblate (English (United States))

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en_US/

Commit:
Translated using Weblate (German)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/de/

Commit:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/pt/

Commit:
Translated using Weblate (Russian)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/ru/

Commit:
Translated using Weblate (French)

Currently translated at 88.5% (402 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/fr/

Commit:
Translated using Weblate (Spanish)

Currently translated at 84.2% (379 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/es/

Commit:
Translated using Weblate (Hebrew)

Currently translated at 70.8% (319 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/he/

Commit:
Translated using Weblate (Tamil)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/ta/

Commit:
Translated using Weblate (English)

Currently translated at 26.2% (119 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en/

Commit:
Translated using Weblate (Japanese)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/ja/

Commit:
Translated using Weblate (Kabyle)

Currently translated at 14.3% (65 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/kab/

Commit:
Translated using Weblate (Japanese)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/ja/

Commit:
Translated using Weblate (German)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/de/

Commit:
Translated using Weblate (Tamil)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/ta/

Commit:
Translated using Weblate (Russian)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/ru/

Commit:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/pt/

Commit:
Translated using Weblate (French)

Currently translated at 94.4% (429 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/fr/

Commit:
Translated using Weblate (Spanish)

Currently translated at 92.6% (417 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/es/

Commit:
Translated using Weblate (Bengali)

Currently translated at 3.7% (17 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/bn/

Commit:
Translated using Weblate (Kabyle)

Currently translated at 20.4% (93 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/kab/

Commit:
Translated using Weblate (English)

Currently translated at 34.8% (158 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en/

Commit:
Translated using Weblate (Hebrew)

Currently translated at 79.7% (359 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/he/

Commit:
Translated using Weblate (Russian)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/ru/

Commit:
Translated using Weblate (German)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/de/

Commit:
Translated using Weblate (Tamil)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/ta/

Commit:
Translated using Weblate (English (United States))

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en_US/

Commit:
Translated using Weblate (Spanish)

Currently translated at 100.0% (450 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/es/

Commit:
Translated using Weblate (Kabyle)

Currently translated at 27.7% (126 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/kab/

Commit:
Translated using Weblate (French)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/fr/

Commit:
Translated using Weblate (Hebrew)

Currently translated at 87.5% (394 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/he/

Commit:
Translated using Weblate (English)

Currently translated at 42.7% (194 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en/

Commit:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/pt/

Commit:
Translated using Weblate (Spanish)

Currently translated at 100.0% (450 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/es/

Commit:
Translated using Weblate (Kabyle)

Currently translated at 27.9% (127 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/kab/

Commit:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/pt/

Commit:
Translated using Weblate (English)

Currently translated at 43.1% (196 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en/

Commit:
Translated using Weblate (German)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/de/

Commit:
Translated using Weblate (Turkish)

Currently translated at 0.2% (1 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/tr/

Commit:
Translated using Weblate (Uzbek)

Currently translated at 2.2% (10 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/uz/

Commit:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/pt/

Commit:
Translated using Weblate (Kabyle)

Currently translated at 31.4% (143 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/kab/

Commit:
Translated using Weblate (Korean)

Currently translated at 9.5% (43 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/ko/

Commit:
Translated using Weblate (Hebrew)

Currently translated at 91.3% (411 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/he/

Commit:
Translated using Weblate (English)

Currently translated at 47.1% (214 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en/

Commit:
Translated using Weblate (English (United States))

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en_US/

Commit:
Added translation using Weblate (Thai)

Commit:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/pt/

Commit:
Translated using Weblate (Kabyle)

Currently translated at 33.7% (153 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/kab/

Commit:
Translated using Weblate (English (United States))

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en_US/

Commit:
Translated using Weblate (Korean)

Currently translated at 14.8% (67 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/ko/

Commit:
Translated using Weblate (Spanish)

Currently translated at 100.0% (450 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/es/

Commit:
Translated using Weblate (Hebrew)

Currently translated at 96.6% (435 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/he/

Commit:
Translated using Weblate (English)

Currently translated at 52.2% (237 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en/

Commit:
Translated using Weblate (French)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/fr/

Commit:
Translated using Weblate (Spanish)

Currently translated at 100.0% (450 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/es/

Commit:
Translated using Weblate (French)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/fr/

Commit:
Translated using Weblate (Kabyle)

Currently translated at 36.3% (165 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/kab/

Commit:
Translated using Weblate (Hebrew)

Currently translated at 100.0% (450 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/he/

Commit:
Translated using Weblate (English)

Currently translated at 55.5% (252 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en/

Commit:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/pt/

Commit:
Translated using Weblate (Korean)

Currently translated at 18.2% (82 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/ko/

Commit:
Translated using Weblate (Hebrew)

Currently translated at 100.0% (450 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/he/

Commit:
Translated using Weblate (Spanish)

Currently translated at 100.0% (450 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/es/

Commit:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/pt/

Commit:
Translated using Weblate (French)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/fr/

Commit:
Translated using Weblate (English (United States))

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en_US/

Commit:
Translated using Weblate (Kabyle)

Currently translated at 38.9% (177 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/kab/

Commit:
Translated using Weblate (Korean)

Currently translated at 22.4% (101 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/ko/

Commit:
Translated using Weblate (English)

Currently translated at 59.2% (269 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en/

Commit:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/pt/

Commit:
Translated using Weblate (Spanish)

Currently translated at 100.0% (450 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/es/

Commit:
Translated using Weblate (English (United States))

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en_US/

Commit:
Translated using Weblate (Kabyle)

Currently translated at 43.1% (196 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/kab/

Commit:
Translated using Weblate (Korean)

Currently translated at 26.4% (119 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/ko/

Commit:
Translated using Weblate (English)

Currently translated at 63.2% (287 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en/

Commit:
Translated using Weblate (Spanish)

Currently translated at 100.0% (450 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/es/

Commit:
Translated using Weblate (English)

Currently translated at 63.8% (290 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en/

Commit:
Translated using Weblate (English (United States))

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en_US/

Commit:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/pt/

Commit:
Translated using Weblate (Kabyle)

Currently translated at 44.9% (204 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/kab/

Commit:
Translated using Weblate (Korean)

Currently translated at 28.4% (128 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/ko/

Commit:
Added translation using Weblate (Italian)

Commit:
Translated using Weblate (Kabyle)

Currently translated at 50.4% (229 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/kab/

Commit:
Translated using Weblate (French)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/fr/

Commit:
Translated using Weblate (Spanish)

Currently translated at 100.0% (450 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/es/

Commit:
Translated using Weblate (English (United States))

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en_US/

Commit:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/pt/

Commit:
Translated using Weblate (Kabyle)

Currently translated at 50.4% (229 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/kab/

Commit:
Translated using Weblate (English)

Currently translated at 67.4% (306 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en/

Commit:
Translated using Weblate (Korean)

Currently translated at 34.0% (153 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/ko/

Commit:
Translated using Weblate (Kabyle)

Currently translated at 51.1% (232 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/kab/

Commit:
Translated using Weblate (English)

Currently translated at 68.0% (309 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en/

Commit:
Translated using Weblate (Korean)

Currently translated at 34.6% (156 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/ko/

Commit:
Translated using Weblate (Tamil)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/ta/

Commit:
Translated using Weblate (Spanish)

Currently translated at 100.0% (450 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/es/

Commit:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/pt/

Commit:
Translated using Weblate (English (United States))

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en_US/

Commit:
Translated using Weblate (Korean)

Currently translated at 42.2% (190 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/ko/

Commit:
Translated using Weblate (English)

Currently translated at 75.3% (342 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en/

Commit:
Translated using Weblate (Kabyle)

Currently translated at 57.2% (260 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/kab/

Commit:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/pt/

Commit:
Translated using Weblate (Spanish)

Currently translated at 100.0% (450 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/es/

Commit:
Translated using Weblate (Kabyle)

Currently translated at 59.6% (271 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/kab/

Commit:
Translated using Weblate (Korean)

Currently translated at 44.8% (202 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/ko/

Commit:
Translated using Weblate (English)

Currently translated at 77.9% (354 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en/

Commit:
Translated using Weblate (Kabyle)

Currently translated at 64.3% (292 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/kab/

Commit:
Translated using Weblate (English)

Currently translated at 83.0% (377 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en/

Commit:
Translated using Weblate (Korean)

Currently translated at 50.2% (226 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/ko/

Commit:
Translated using Weblate (Spanish)

Currently translated at 100.0% (450 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/es/

Commit:
Translated using Weblate (English (United States))

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en_US/

Commit:
Translated using Weblate (Kabyle)

Currently translated at 69.6% (316 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/kab/

Commit:
Translated using Weblate (English)

Currently translated at 88.5% (402 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en/

Commit:
Translated using Weblate (Korean)

Currently translated at 56.2% (253 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/ko/

Commit:
Translated using Weblate (English (United States))

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en_US/

Commit:
Translated using Weblate (English)

Currently translated at 90.7% (412 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en/

Commit:
Translated using Weblate (Korean)

Currently translated at 58.6% (264 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/ko/

Commit:
Translated using Weblate (Kabyle)

Currently translated at 72.0% (327 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/kab/

Commit:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/pt/

Commit:
Translated using Weblate (Kabyle)

Currently translated at 83.7% (380 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/kab/

Commit:
Translated using Weblate (Korean)

Currently translated at 73.1% (329 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/ko/

Commit:
Translated using Weblate (English)

Currently translated at 93.1% (423 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en/

Commit:
Translated using Weblate (English (United States))

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en_US/

Commit:
Translated using Weblate (French)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/fr/

Commit:
Translated using Weblate (English)

Currently translated at 93.8% (426 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en/

Commit:
Translated using Weblate (Spanish)

Currently translated at 100.0% (450 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/es/

Commit:
Translated using Weblate (Kabyle)

Currently translated at 86.3% (392 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/kab/

Commit:
Translated using Weblate (Korean)

Currently translated at 75.5% (340 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/ko/

Commit:
Translated using Weblate (French)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/fr/

Commit:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/pt/

Commit:
Translated using Weblate (Kabyle)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/kab/

Commit:
Translated using Weblate (English)

Currently translated at 100.0% (454 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/en/

Commit:
Translated using Weblate (Korean)

Currently translated at 100.0% (450 of 450 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/ko/

Commit:
Translated using Weblate (Arabic)

Currently translated at 6.3% (29 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/ar/

Commit:
Translated using Weblate (Bengali)

Currently translated at 6.6% (30 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/bn/

Commit:
Translated using Weblate (Italian)

Currently translated at 6.3% (29 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/it/

Commit:
Translated using Weblate (Persian)

Currently translated at 6.3% (29 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/fa/

Commit:
Translated using Weblate (Thai)

Currently translated at 6.3% (29 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/th/

Commit:
Translated using Weblate (Uzbek)

Currently translated at 8.1% (37 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/uz/

Commit:
Translated using Weblate (Vietnamese)

Currently translated at 8.3% (38 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/vi/

Commit:
Translated using Weblate (Turkish)

Currently translated at 6.1% (28 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/tr/

Commit:
Added translation using Weblate (Hindi)

Commit:
Translated using Weblate (Arabic)

Currently translated at 7.9% (36 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/ar/

Commit:
Translated using Weblate (Italian)

Currently translated at 7.9% (36 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/it/

Commit:
Translated using Weblate (Bengali)

Currently translated at 8.1% (37 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/bn/

Commit:
Translated using Weblate (Persian)

Currently translated at 7.9% (36 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/fa/

Commit:
Translated using Weblate (Thai)

Currently translated at 7.9% (36 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/th/

Commit:
Translated using Weblate (Turkish)

Currently translated at 7.7% (35 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/tr/

Commit:
Translated using Weblate (Uzbek)

Currently translated at 9.6% (44 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/uz/

Commit:
Translated using Weblate (Vietnamese)

Currently translated at 9.9% (45 of 454 strings)

Translation: Umi-OCR/UI
Translate-URL: https://hosted.weblate.org/projects/umi-ocr/ui/vi/
